dart/biomechanics/MarkerFitter.cpp

```diff
@@ -588,7 +594,7 @@ MarkerFitter::MarkerFitter(
     mSilenceOutput(false),
     mDisableLinesearch(false),
     mAnatomicalMarkerDefaultWeight(1.0),
-    mTrackingMarkerDefaultWeight(0.02),
+    mTrackingMarkerDefaultWeight(0.2),
```

Comment:
In the final problem run by `engine.py`, this value is set to 0.05. Does this change the default for that problem too?

Reply:
No, this only changes the default if one isn't set. For the initial IK scale+fit problems, I think(?) it's not set? It's at least not set on the C++ test harness. If `engine.py` sets this value, then changing this default will have had no effect. If `engine.py` sets this value, that could also explain differences between the C++ test results and Python...

Reply:
Just to clarify, in the `MarkerFitter` problem configuration in `engine.py`, the following is set: `markerFitter.setRegularizeTrackingMarkerOffsets(0.05)`.  But that is the full optimization, not the scale + IK, I think? I think we're good here, just double checking.